webhook/resourcesemantics/validation/controller.go

```diff
@@ -40,17 +40,32 @@ func NewAdmissionController(
 	handlers map[schema.GroupVersionKind]resourcesemantics.GenericCRD,
 	wc func(context.Context) context.Context,
 	disallowUnknownFields bool,
+	callbacks ...map[schema.GroupVersionKind]Callback,
 ) *controller.Impl {
 
 	client := kubeclient.Get(ctx)
 	vwhInformer := vwhinformer.Get(ctx)
 	secretInformer := secretinformer.Get(ctx)
 	options := webhook.GetOptions(ctx)
 
+	// This not ideal, we are using a variadic argument to effectively make callbacks optional
+	// This allows this addition to be non-breaking to consumers of /pkg
+	// TODO: once all sub-repos have adoped this, we might move this back to a traditional param.
+	if len(callbacks) > 1 {
+		panic("NewAdmissionController may not be called with multiple callback maps")
+	}
+	unwrappedCallbacks := map[schema.GroupVersionKind]Callback{}
+	for _, m := range callbacks {
+		for k, v := range m {
+			unwrappedCallbacks[k] = v
+		}
+	}
+
```

Comment:
we can drop the loops and just check for `len(callbacks) == 1`

Reply:
Done!